packages/nimble-components/src/table/template.ts

```diff
@@ -145,7 +145,7 @@ export const template = html<Table>`
                             ${repeat(x => x.virtualizer.visibleItems, html<VirtualItem<HTMLElement>, Table>`
                                 ${when((x, c: ExecutionContext<Table>) => c.parent.tableData[x.index]?.isGroupRow, html<VirtualItem<HTMLElement>, Table>`
                                     <${tableGroupRowTag}
-                                        class="group-row"
+                                        class="group-row ${(_, c) => (c.parent.virtualizer.isScrolling ? '' : 'allow-hover')}"
```

Comment:
I'm wondering if adding this state with class is the approach we want. I can't think of any other place where we add a CSS class to a child element and then put the styling associated with that CSS class in the child element's styles. Generally, if we want state to be passed to the child, we use an attribute. Is there a reason to use a CSS class instead here?